apps/site/src/lib/styles/_bootstrap-utils.scss

```diff
@@ -9,3 +9,6 @@
 $font-family-base: var(--next-font-fuzzy-bubbles);
 $font-size-base: 1.25rem;
 $headings-font-weight: 700;
+
+// $accordion-button-icon: url("~@/assets/icons/plus-lg.svg");
+// $accordion-button-active-icon: url("~@/assets/icons/dash-lg.svg");
```

Comment:
Once you download the icons from Figma, uncommenting this code should replace the defaults properly.